packages/account/src/Sections/Security/Passkeys/passkeys.scss

```diff
@@ -43,6 +43,19 @@
                 gap: 0.8rem;
                 border-bottom: 1px solid var(--general-hover);
                 padding: 1.6rem 0;
+
+                ul {
+                    list-style: disc;
```

Comment:
Isn't `disc` the default type of `ul` tag?

Reply:
it was no there, bro. 
it might be some overwritten rules.. I didn't check it